src/component/schema.ts

```
@@ -1,16 +1,140 @@
 import { defineSchema, defineTable } from "convex/server";
-import { ObjectType, v } from "convex/values";
+import { v } from "convex/values";
 
+// https://docs.expo.dev/push-notifications/sending-notifications/#message-request-format
 export const notificationFields = {
+  _contentAvailable: v.optional(v.boolean()),
+  data: v.optional(v.any()),
   title: v.string(),
-  subtitle: v.optional(v.string()),
   body: v.optional(v.string()),
-  sound: v.optional(v.string()),
-  data: v.optional(v.any()),
-  categoryIdentifier: v.optional(v.string()),
```

If folks already have this data in their database we'll need to make a version with a migration path to move it over, but it'll fail at push time so they'll know they need to fix it (it won't silently fail unless they've disabled schema validation